### DISCUSSION TITLE
Question about f1 data & mini sectors

### DISCUSSION THREAD

**Discussion by SyntaxSnipes** [slowlydev/f1-dash#150] [Q&A] [answered]
Hi, where do you get the mini-sector data from? And how do you get them live??

**Comment by slowlydev** [comment, marked as answer]
From the f1 realtime api. And the mini-sectors are one of the topics we can subscribe to and get updates about their state (fastest, slower, pits, pb).

Edit:
Here we connect: https://github.com/slowlydev/f1-dash/blob/main/crates/client/src/client.rs
Here we have the topics and urls: https://github.com/slowlydev/f1-dash/blob/main/crates/client/src/consts.rs

**Reply by SyntaxSnipes** [reply]
what do you mean by subscribe? i have to pay?? if so where? thanks

**Reply by slowlydev** [reply]
No no. no payment is involved. Like you subscribe to a topic and its updates.
